src/metadata/mwdb_uploads.py

```diff
@@ -0,0 +1,64 @@
+import re
+import urllib.parse
+
+from metadata import Metadata
+from mwdblib import Malwarecage  # type: ignore
```

Comment:
I *think* you should add
```
[mypy-mwdblib.*]
ignore_missing_imports = True
```
to setup.cfg instead, and remove the `type: ignore` (@BonusPlay removed all type:ignores from the project that way)

Reply:
Done :+1: